test/test-format.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import test from 'ava';
+
+import {Options} from '../src/cli';
+import * as format from '../src/format';
+import {nop} from '../src/util';
+
+import {withFixtures} from './fixtures';
+
+const OPTIONS: Options = {
+  gtsRootDir: './',
+  targetRootDir: './',
+  dryRun: false,
+  yes: false,
+  no: false,
+  logger: {log: console.log, error: console.error, dir: nop}
+};
+
+test.serial('format should return false for ill-formatted files', async t => {
+  await withFixtures(
+      {
+        'tsconfig.json': JSON.stringify({files: ['a.ts']}),
+        'a.ts': 'export const foo = 2;  '
```

Comment:
Is this ill-formatted because of the trailing spaces? Can you add a comment for that so that future readers don't accidentally delete the spaces?

Reply:
Actually I should've put this comment in line 51 below because if someone accidentally removes the trailing spaces there, the test will still pass.

Reply:
I'd put the same comment here too so that people can easily understand why this line must fail :)